app/presenters/batch_presenter.rb

```diff
@@ -8,7 +8,8 @@ class BatchPresenter
 
   ##
   # Job Statuses
-  JOB_STATUSES = { unavailable: 'Unavailable'.freeze,
+  JOB_STATUSES = { new:         'New'.freeze,
+                   unavailable: 'Unavailable'.freeze,
```

Comment:
Is unavailable like failed?

Reply:
It means the status has expired, or otherwise that jobs are not enqueued. This status existed previously.

It's possible for XML Import batches to find their way to `Unavailable` if a file is added but no jobs are enqueued.